File: Math/Least Common Multiple/lcm.py
def gcd(a, b): # Greatest Common Divisor
  if b == 0:
    return a
  
  return gcd(b, a % b)

def lcm(a, b):
  return a // gcd(a, b) * b

a, b = map(int, input().split())
print(lcm(a, b))

File: Math/Least Common Multiple/lcm.rb
def gcd a, b # Geatest Common Divisor
  if b == 0
    a
  else
    gcd b, a % b
  end
end

def lcm a, b
  a / gcd(a, b) * b
end

a, b = gets.split.map(&:to_i)
puts lcm a, b
